lib/pymedphys/_experimental/chartchecks/helpers.py

```diff
@@ -168,6 +169,16 @@ def get_all_dicom_treatment_info(dicomFile):
             except (TypeError, ValueError, AttributeError):
                 dicomBeam["tolerance"] = 0
 
+            if dicomBeam["manufacturer"] == "Varian":
+
+                dicomBeam["gantry_angle"] = (180 - dicomBeam["gantry_angle"]) % 360
+                dicomBeam["collimator_angle"] = (
+                    180 - dicomBeam["collimator_angle"]
+                ) % 360
+                dicomBeam["couch_ang"] = (180 - dicomBeam["couch_ang"]) % 360
```

Comment:
These lines are all duplicating the same work. When this happens, pull the logic out into a separate function and then call that. For example:

```python
def _invert_angle(angle):
    return (180 - angle) % 360


for key in ["gantry_angle", "collimator_angle", "couch_ang"]:
    dicomBeam[key] = _invert_angle(dicomBeam[key])
```

Also, if you change "couch_ang" -> "couch_angle" to be consistent with your other naming conventions you can change the above to:

```python

angle_keys = [key for key in dicomBeam.keys() if "angle" in key]

for key in angle_keys:
    dicomBeam[key] = _invert_angle(dicomBeam[key])
```

That second version has the benefit of also working should you add another "angle" key that also needs transformation.

Also :), change the variable name `dicomBeam` to `dicom_beam` (snake case).